src/Azure.Functions.Cli/ExtensionBundle/ExtensionBundleConfigurationBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Azure.Functions.Cli.Common;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Azure.Functions.Cli.ExtensionBundle
+{
+    internal class ExtensionBundleConfigurationBuilder : IConfigureBuilder<IConfigurationBuilder>
+    {
+        private readonly ScriptApplicationHostOptions _hostOptions;
+
+        public ExtensionBundleConfigurationBuilder(ScriptApplicationHostOptions hostOptions)
+        {
+            _hostOptions = hostOptions;
+        }
+
+        public void Configure(IConfigurationBuilder builder)
+        {
+            var bundleId = ExtensionBundleHelper.GetExtensionBundleOptions(_hostOptions).Id;
+            if (!string.IsNullOrEmpty(bundleId))
+            {
+                builder.AddInMemoryCollection(new Dictionary<string, string>
+                {
+                    { "AzureFunctionsJobHost:extensionBundle:downloadPath", Path.Combine(Path.GetTempPath(), "Functions", ScriptConstants.ExtensionBundleDirectory, bundleId)}
```

Comment:
will this work on linux?

Reply:
as long as the tempPath can resolve it should work.